apps/api/src/routes/applicationRouter.ts

```diff
@@ -350,6 +350,61 @@ applicationRouter.post(
 	),
 );
 
+applicationRouter.post('/applications/:applicationId/revoke', jsonParser, async (request: Request, response: Response) => {
```

Comment:
I believe we have moved all the `/applications` paths out of this router, can we make this consistent with the other endpoints? (also needs to be removed from /close).